tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests.csproj

```diff
@@ -16,11 +16,14 @@
     <PackageReference Include="nunit" Version="3.12.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="3.1.3" />
```

Comment:
Still required?

Reply:
Nope InMemory does not support Transactons, removed